src/table/utils/__tests__/handle-key-press.spec.js

```diff
@@ -457,8 +457,8 @@ describe('handle-key-press', () => {
         announce,
         keyboard,
       });
-      expect(evt.preventDefault).not.toHaveBeenCalledTimes(1);
-      expect(evt.stopPropagation).not.toHaveBeenCalledTimes(1);
+      expect(evt.preventDefault).not.toHaveBeenCalled();
```

Comment:
isn't this less specific?

Reply:
if it calls two times, `not.toHaveBeenCalledTimes(1)` is still true, right?

Reply:
ah sorry it is a `.not.` check, yes your change is better